From f78f934710394822a36bd74043ed93a812c1c690 Mon Sep 17 00:00:00 2001
From: Robert Marko <robert.marko@sartura.hr>
Date: Mon, 12 Jan 2026 15:36:35 +0100
Subject: [PATCH] microchip: lan969x: do not rely on Ruby shebang

Host Ruby build in the staging dir must be used, so we cannot rely on the
shebang as that will fail.

So, call the script via Ruby executable instead.

Signed-off-by: Robert Marko <robert.marko@sartura.hr>
---
 plat/microchip/lan969x/common/common.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/plat/microchip/lan969x/common/common.mk
+++ b/plat/microchip/lan969x/common/common.mk
@@ -218,9 +218,9 @@ FWU_HTML := ${BUILD_PLAT}/fwu.html
 FWU_JS   := ${BUILD_PLAT}/fwu_app.js
 
 ${FWU_JS}: ${BUILD_PLAT}/${FWU_FIP_NAME}
-	./plat/microchip/scripts/mkjs.rb -p ${PLAT} -o ${FWU_JS} $<
+	$(Q)ruby ./plat/microchip/scripts/mkjs.rb -p ${PLAT} -o ${FWU_JS} $<
 
 ${FWU_HTML}: ${FWU_JS}
-	./plat/microchip/scripts/html_inline.rb -i ${BUILD_PLAT} ./scripts/fwu/serial.html > ${FWU_HTML}
+	$(Q)ruby ./plat/microchip/scripts/html_inline.rb -i ${BUILD_PLAT} ./scripts/fwu/serial.html > ${FWU_HTML}
 
 all: ${FWU_HTML}
